lib/sles4sap_publiccloud.pm

```diff
@@ -333,7 +333,7 @@ sub stop_hana {
         # It is better to wait till ssh disappear
         record_info("Wait ssh disappear start");
         my $out = $self->{my_instance}->wait_for_ssh(timeout => 60, wait_stop => 1);
-        record_info("Wait ssh disappear end", "$out") if (defined $out);
+        record_info("Wait ssh disappear end", "out:" . ($out // 'undefined'));
```

Comment:
Without `(` `)` the precedence is not the intended one and it result in http://openqaworker15.qa.suse.cz/tests/280302#step/Crash_site_a-primary/647